### ISSUE TITLE
Retiring jenkins robot job

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#1043]
We have a Jenkins server here:

https://build.obolibrary.io/job/ontodev/job/robot/

AFAICR this was for building jars for branches in robot

We are thinking of shutting down this machine (@kltm). I am not sure the functionality of this jenkins job has been used in years. I think perhaps the better protocol is to make a pre-release from a branch

however we can figure a solution to keep it running if required

**Comment by jamesaoverton** [comment]
I have no objections to shutting down the machine. Please just give me some warning.

We were using that Jenkins machine to build and archive JARs for ROBOT feature branches and pre-releases. It also uses the branch JAR to build OBI, which is a slightly more intense test than our GitHub CI, but hasn't revealed any ROBOT problems (ever?).

I see 21 hits on GitHub for code containing 'build.obolibrary.io': <https://github.com/search?q=%22build.obolibrary.io%22&type=code>. Most of them are mine and I should just update them to use proper ROBOT releases.

Now we use GitHub Actions and attach JARs to every run, which is pretty much equivalent to what we use Jenkins. The main advantage of Jenkins is that you can't seem to get a proper URL for the artifacts attached to a GitHub Actions run: You have to use a browser to download them, because of some client-side magic. (Let me know if I missed something.)

(Actually there was a bug in the GitHub Action so artifacts were not being uploaded, which I just fixed with <https://github.com/ontodev/robot/commit/e2aaf880ba3a3a255cf5af6525175ad212676bd8>.)

We also make GitHub pre-releases, as you suggested.

**Comment by kltm** [comment]
We can be very flexible here and if we want to cost and maintain this long-term, we can revisit. But for now, how about we say that we'd like to suspend the server on September 31st?

If you're looking for stable URLs from GHA, I'd also suggest using an AWS S3 push to make that happen. It would certainly be more stable than Jenkins.

Tagging @jamesaoverton @matentzn

**Comment by kltm** [comment]
@jamesaoverton @matentzn I wanted to ping on this again as we are two weeks out from the discussed server suspension. Is there anything we can do to help support the suspension of this machine?

**Comment by kltm** [comment]
Note from @jamesaoverton on slack that we are on track for the end of the month and that we should proceed accordingly.

**Comment by kltm** [comment]
@jamesaoverton @matentzn (and @cmungall ) Shall we go ahead and turn off this machine. I lost track of our timeline and we are over time. Is there anything else we want to do here?

**Comment by jamesaoverton** [comment]
Yes, go ahead and turn it off. You've been more than patient! :smile:

**Comment by kltm** [comment]
All services on the system are stopped. I'll leave it like that for a little and, if there is no complain or regret, will pull the plug on the machine.
